samples/charts/financial-chart/indicator-types/Pages/FinancialChartIndicatorTypes.razor

```diff
@@ -26,6 +26,21 @@
 @code {
 
     List<StockItem> DataSource;
+    public FinancialChart financialChart;
+    public FinancialChart FinancialChartRef
+    {
+        get
+        {
+            return financialChart;
+        }
+        set
+        {
+            financialChart = value;
+            this.FinancialChartRef.IndicatorTypes.Clear();
+            this.FinancialChartRef.IndicatorTypes.Add(FinancialIndicatorType.MovingAverageConvergenceDivergence);
+            this.FinancialChartRef.IndicatorTypes.Add(FinancialIndicatorType.RelativeStrengthIndex);
+        }
```

Comment:
@mddifilippo89 you need to report this as bug with info about not able to set IndicatorTypes property in markup code